xml/obs_scm_ci_workflow_integration.xml

```diff
@@ -983,6 +983,90 @@ test_build:
       </note>
     </sect2>
 
+    <sect2 xml:id="sec.obs.obs_scm_ci_workflow_integration.setup.event_action_and_status_report_table">
```

Comment:
I think this would better fit into it's own section (`sect1`). I would put it between `SCM/CI Workflow Integration Setup` and `SCM/CI Workflow Versions`